src/components/panes/declared-variables-pane.jsx

```diff
@@ -21,7 +23,9 @@ export class DeclaredVariablesPaneUnconnected extends React.Component {
     return (
       <SidePane task={tasks.toggleDeclaredVariablesPane} title="Declared Variables" openOnMode="declared variables">
         <div className="declared-variables-list">
-          {this.props.userDefinedVarNames.map(varName =>
+          {this.props.userDefinedVarNames
+            .filter(varName => !doNotShow.has(varName))
+          .map(varName =>
```

Comment:
see other comment -- we filter out `initialVariables` in notebook-reducer, so i think maybe no changes are needed in this file?

Reply:
If this were true, `CodeMirror` would not be in the declared variables pane by default, right? Perhaps the available globals are getting checked before we instantiate CodeMirror though.

Reply:
fixed - thx for feedback!